pi4j-test/src/test/java/com/pi4j/test/platform/ManualPlatformsTest.java

```diff
@@ -39,18 +42,18 @@
 import com.pi4j.plugin.mock.provider.pwm.MockPwmProvider;
 import com.pi4j.plugin.mock.provider.serial.MockSerialProvider;
 import com.pi4j.plugin.mock.provider.spi.MockSpiProvider;
-import org.junit.After;
-import org.junit.Before;
-import org.junit.Test;
-
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.TestInstance.Lifecycle;
 
+@TestInstance(Lifecycle.PER_CLASS)
 public class ManualPlatformsTest {
 
     private Context pi4j;
 
-    @Before
+    @BeforeAll
```

Comment:
This won't work, as @BeforeAll must be a static method:

https://junit.org/junit5/docs/current/user-guide/#writing-tests-annotations

Reply:
yes and no ;-)
by adding @TestInstance(Lifecycle.PER_CLASS) to the class, it is not needed to be static. 
static would also break the reference to private Context pi4j

Reply:
oh my... you are right. Sorry.